docs/UserGuide.md

```diff
@@ -230,35 +239,33 @@ Examples:
 
 ### Add / Clear pets' appointment details: `app` ###
 
-Sometimes, you will need to bring the pet out for its vet or grooming appointments. You will want to put a reminder on
-the pet to take note as well. With this feature you can **_Add Appointment_** and **_Clear Appointment_** when appropriate.
+In the event that your client tasks you with bringing their pets out for their vet or grooming appointments, you may wish 
+to note down the relevant details of such appointments, such as the date, time and location.
 
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about this feature:**<br>
 
-* You can only store one appointment to a pet at any point of time.
-  * This is because pets often have varying appointment dates, and it is unlikely to know when the next appointment will be.
-* Once the appointment is over, you have to clear it if you want to put new appointments in the future.
+* You can only note down one appointment per pet at any point in time.
+* Once the appointment is over, you will have to clear it manually if you wish to note down a new appointments.
 
 </div>
 
 **Add Appointment**
 
 Format: `app INDEX dt/[dd-MM-yyyy HH:mm] at/[location]`
 
-* Adds appointment for pet at `INDEX` on `dt/[dd-MM-yyyy HH:mm]` `at/[location]` .
+* Adds appointment for pet at `INDEX` on a specific date at a specific location.
 * The index refers to the index number shown in the current list of pets.
 * The index **must be a positive integer** 1, 2, 3, …​
-* Date and time of appointment should be entered together with `dt/` prefix.
+* Date and time of the appointment should be entered after the `dt/` prefix.
 * Date and time should strictly follow `dd-MM-yyyy HH:mm` format.
-* Location of appointment should be entered with `at/` prefix.
+* Location of appointment should be entered after the `at/` prefix.
 * Whitespaces, special characters and alphanumeric characters are allowed for location.
-* If both `date/` and `at/` are not present, `app` will be deemed invalid.
+* Both `date/` and `at/` are **mandatory** fields.
```

Comment:
```suggestion
* Both `dt/` and `at/` are **mandatory** fields.
```
Did not notice this initially when I did it

Reply:
Thanks for pointing it out!